<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TegraRcmGUI</class>
 <widget class="QMainWindow" name="TegraRcmGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>428</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>690</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TegraRcmGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>481</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QFrame" name="devStatusFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px; 
background-color: rgb(0, 150, 136);
border-color: rgb(0, 0, 0);
QLabel{font: 75 11pt &quot;Calibri&quot;; color: rgb(255, 255, 255);}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="devStatusLbl_2">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="rcmStatusFrame">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>55</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px; 
background-color: rgb(0, 150, 136);
border-color: rgb(0, 0, 0);
QLabel{font: 75 11pt &quot;Calibri&quot;; color: rgb(255, 255, 255);}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="rcmStatusLbl_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="arianeStatusFrame">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>105</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px; 
background-color: rgb(0, 150, 136);
border-color: rgb(0, 0, 0);
QLabel{font: 75 11pt &quot;Calibri&quot;; color: rgb(255, 255, 255);}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="arianeStatusLbl_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QFrame" name="deviceInfoFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="deviceInfoLbl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Calibri&quot;;</string>
     </property>
     <property name="text">
      <string>DEVICE INFO</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="statusBoxFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="rcmStatusLbl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>15</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>RCM status:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="autoLaunchArianeLbl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Ariane autoboot:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="arianeStatusLbl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>65</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ariane status:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pushLayoutWidget" native="true">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>311</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0)</string>
    </property>
    <layout class="QVBoxLayout" name="pushLayout">
     <item>
      <layout class="QVBoxLayout" name="push_layout"/>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="titleBarFrame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>691</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="titleBarLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;Rubik&quot;;</string>
     </property>
     <property name="text">
      <string>TegraRcmGUI v3.0-a by eliboa</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="closeAppBtn">
     <property name="geometry">
      <rect>
       <x>665</x>
       <y>0</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="MoveWindowWidget" name="moveWindowWidget" native="true">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>641</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="minimizeAppBtn">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>0</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="deviceInfoBoxFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>181</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="devInfoDisableLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable Ariane autoboot 
 to display device info</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="di_tableWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>15</y>
       <width>171</width>
       <height>220</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>18</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>18</number>
     </attribute>
    </widget>
    <zorder>di_tableWidget</zorder>
    <zorder>devInfoDisableLbl</zorder>
   </widget>
   <zorder>statusBoxFrame</zorder>
   <zorder>tabWidget</zorder>
   <zorder>devStatusFrame</zorder>
   <zorder>rcmStatusFrame</zorder>
   <zorder>arianeStatusFrame</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>pushLayoutWidget</zorder>
   <zorder>titleBarFrame</zorder>
   <zorder>deviceInfoBoxFrame</zorder>
   <zorder>deviceInfoFrame</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MoveWindowWidget</class>
   <extends>QWidget</extends>
   <header>qutils.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
